content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -89,7 +89,7 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the sof
 
 10. Build `rippled` binary executable from source code. This may take about 30 minutes, depending on your hardware specs.
```

Comment:
Extra step you may want to put in between 9 and 10 here. If you previously built, or (more importantly) tried and failed to build `rippled`, you should delete the `build/` directory to start clean. Otherwise, you may get unexpected behavior like a `rippled` executable that crashes due to segfaults.

If this is your first time building `rippled`, you won't have a `build/` directory and can skip that step.